beacon-chain/slasher/helpers.go

```diff
@@ -0,0 +1,53 @@
+package slasher
```

Comment:
Helpers file created to add various pure functions used in slashing detection to a single, concise place